### PR TITLE
update release/2.4.0

### PR DESCRIPTION
**Pull request by laughingman-hass**
https://app-updates.agilebits.com/product_history/SCIM#v204002

### CODE REVIEW

File: beta/workspace-settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "actor":"admin.email.goes.here@example.com",
+    "domain":"example.1password.om",
```
Comment (ag-adampike) [inline beta/workspace-settings.json:3]: Small typo here. I assume `example.1password.com` was intended.

Perhaps some commments would also be helpful here throughout, e.g.:
```
    "domain":"example.1password.com",            // the sign-in address of your 1Password account
```

File: docker/README.md [hunk 2]
```diff
@@ -82,10 +82,27 @@ You’ll also need to set the environment variable `OP_LETSENCRYPT_DOMAIN` withi
 
 Ensure that `OP_LETSENCRYPT_DOMAIN` is set to the domain name you’ve set up before continuing.
 
+(Optional) If you’re part of the Google Workspace provisioning beta, you will need to set up some additional secrets to use this functionality. Refer to our complete Google Workspace provisioning beta documentation for more complete information.
+
+First, you will want to edit the file located at `scim-examples/beta/workspace-settings.json` and enter in the appropriate details.
```
Comment (ag-adampike) [inline docker/README.md:87]: For clarity and readability, I would suggest removing "you will want to" and other similar phrases throughout.

File: docker/README.md [hunk 3]
```diff
@@ -101,6 +118,20 @@ Unlike Docker Compose, you won’t need to set the `OP_SESSION` variable in `sci
 
 You’ll still need to set the environment variable `OP_LETSENCRYPT_DOMAIN` within `scim.env` to the URL you selected during [PREPARATION.md](/PREPARATION.md). Open that in your preferred text editor and change `OP_LETSENCRYPT_DOMAIN` to that domain name.
 
+(Optional) If you’re part of the Google Workspace provisioning beta, you will need to set up some additional secrets to use this functionality. Refer to our complete Google Workspace provisioning beta documentation for more complete information.
```
Comment (ag-adampike) [inline docker/README.md:121]: Just a suggestion: can we put the optional sections under a collapsible heading? e.g.:

<details>
  <summary> Information for Google Workspace beta participants </summary>
  
  ## Google Workspace beta instructions
  
  First, edit the file located at `scim-examples/beta/workspace-settings.json` and enter in the appropriate details.
…
</details>

**Review by ag-adampike — CHANGES_REQUESTED** [review]
I don't have a great way to test the Google Workspace integration atm, so I can't fully test, but other than the small comments I added below, LGTM!!

File: docker/README.md (hunk 3 above)
Comment (laughingman-hass) [reply docker/README.md:121]: I like this suggestion

**Review by ag-adampike — APPROVED** [review]
LGTM!🎉

**Review by grellyd — APPROVED** [review]
Looks good to me. Given Adam already took it all for a spin, I have not. Let me know if you would like a second round to tests!

**Pull request merged**